ui/app/src/context/Config.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { createContext, useContext, useMemo } from 'react';
+import { marked } from 'marked';
+import * as DOMPurify from 'dompurify';
+import { ConfigModel, useConfig } from '../model/config-client';
+
+interface ConfigContextType {
+  config: ConfigModel;
+}
+
+const ConfigContext = createContext<ConfigContextType | undefined>(undefined);
+
+export function ConfigContextProvider(props: { children: React.ReactNode }) {
+  const { data, isLoading } = useConfig();
+  if (isLoading || data === undefined) {
+    return <></>;
```

Comment:
```suggestion
    return <Stack width="100%" sx={{ alignItems: 'center', justifyContent: 'center' }}>
        <CircularProgress />
      </Stack>;
```

Reply:
okey. I just wished we would have the Perses logo animated instead of the MUI spinner 😢